src/config.c

```diff
@@ -38,3 +38,27 @@ long long Config_GetThreadCount(RedisModuleCtx *ctx, RedisModuleString **argv, i
 
 	return threadCount;
 }
+
+uint64_t Config_GetEntitiesThreshold(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
This is an implicit cast from `long long` to `uint64_t` that I find mildly concerning. I suggest either casting explicitly or make `entities_threshold` a `long long` everywhere.

Reply:
explicitly casted